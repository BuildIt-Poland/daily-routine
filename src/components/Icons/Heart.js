import React from 'react';

function Heart() {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="17"
      height="18"
      viewBox="0 0 17 18"
      aria-labelledby="buildit-logo-title"
    >
      <title id="buildit-logo-title">LOVE</title>
      <path
        fill="#222"
        d="M8.5 17.5S.113 12.504.113 6.694a4.194 4.194 0 1 1 8.387 0 4.194 4.194 0 0 1 8.388 0C16.887 12.504 8.5 17.5 8.5 17.5"
      />
    </svg>
  );
}

export default Heart;
