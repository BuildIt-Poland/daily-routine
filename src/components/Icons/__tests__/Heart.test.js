import React from 'react';
import { create } from 'react-test-renderer';

import Heart from '../Heart';

describe('COMPONENT - Icons Heart', () => {
  it('renders correctly', () => {
    const component = create(<Heart />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
