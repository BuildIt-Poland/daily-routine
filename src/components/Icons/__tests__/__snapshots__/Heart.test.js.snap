// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Icons Heart renders correctly 1`] = `
<svg
  aria-labelledby="buildit-logo-title"
  height="16"
  viewBox="0 0 17 16"
  width="17"
  xmlns="http://www.w3.org/2000/svg"
>
  <title
    id="buildit-logo-title"
  >
    LOVE
  </title>
  <path
    d="M8.5 15.5S.113 10.504.113 4.694a4.194 4.194 0 1 1 8.387 0 4.194 4.194 0 0 1 8.388 0C16.887 10.504 8.5 15.5 8.5 15.5"
    fill="currentColor"
  />
</svg>
`;
